.coveragerc

```diff
@@ -0,0 +1,6 @@
+[report]
```

Comment:
this is what I normally include:

```
omit =
    */bin/test
    */eggs/*
    */lib/python*
    */parts/*
    */usr/*
```

I don't think is a good idea to omit tests or upgrade steps code.